pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -807,6 +815,12 @@ async def _responses_create(
             openai_messages.insert(0, responses.EasyInputMessageParam(role='system', content=instructions))
             instructions = NOT_GIVEN
 
+        if verbosity := model_settings.get('openai_text_verbosity'):
+            if text is None:
+                text = {'format': {'type': 'text'}, 'verbosity': verbosity}
+            else:
+                text['verbosity'] = verbosity
```

Comment:
I don't think the `'format': {'type': 'text'}` is required is it?

```suggestion
        if verbosity := model_settings.get('openai_text_verbosity'):
            text = text or {}
            text['verbosity'] = verbosity
```